src/xdocs/releasenotes_old.xml

```diff
@@ -274,8 +274,8 @@
         </li>
         <li>
           Enhanced the property types
-          <a href="property_types.html#stringSet">stringSet</a> and
-          <a href="property_types.html#intSet">intSet</a> to support being
+          <a href="property_types.html#String[]">String[]</a> and
```

Comment:
please encode `[]`  at `#String[]` links should work in browser.

Reply:
Sorry, my bad, done.